doc/en/apiref.rst

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
git has no idea about renames, it tracks plain snapshots without any more metadata